### DISCUSSION TITLE
Export Action logs me out of the application

### DISCUSSION THREAD

**Discussion by hrodrigues-belodigital** [filamentphp/filament#13507] [Help]
### Package

Actions

### Package Version

v3

### How can we help you?

Hello,

I've been using filamentphp for a while now and I've been now working with export actions on my tables.

When I'm using an export action and I try to export data from a table, it used to expired my session and log me out of my app. I fixed the part where it would expire my session but now when I try to export, it just logs me out of my application and when I log in again, the exported data is in the notification bell is ready to download.

Does anyone know why is this happening and how can I fix this issue?

I thought it could be due to large chunks of data but it also happens when exporting for example 10 records (not consistent but sometimes it happens).

Can anyone help me?

**Comment by izzudin96** [comment]
Do you have any listener that listen to Login events? 

Check if `vendor/filament/actions/src/Exports/Jobs/ExportCsv.php` -> `handle()` trips you.

**Reply by hrodrigues-belodigital** [reply]
Hey. I ended up fixing it. Apparently it was due to the SESSION_DRIVER env variable.
